RepoDb/RepoDb/QueryField.cs

```diff
@@ -209,23 +209,20 @@ public override string ToString()
             }
 
             // Name
-            var name = expression.GetName();
-            if (PropertyCache.Get<TEntity>().Any(property => string.Equals(PropertyMappedNameCache.Get(property.PropertyInfo), name, StringComparison.OrdinalIgnoreCase)) == false)
+            var field = expression.GetField();
+            if (PropertyCache.Get<TEntity>().Any(property => string.Equals(PropertyMappedNameCache.Get(property.PropertyInfo), field.Name, StringComparison.OrdinalIgnoreCase)) == false)
             {
-                throw new InvalidExpressionException($"Invalid expression '{expression.ToString()}'. The property {name} is not defined on a target type '{typeof(TEntity).FullName}'.");
+                throw new InvalidExpressionException($"Invalid expression '{expression.ToString()}'. The property {field.Name} is not defined on a target type '{typeof(TEntity).FullName}'.");
             }
 
-            // Type
-            var type = expression.GetMemberType();
-
             // Value
             var value = expression.GetValue();
 
             // Operation
             var operation = GetOperation(expression.NodeType);
 
             // Return the value
-            var field = new Field(name, type);
+            //var field = new Field(name, type);
```

Comment:
Just remove this commented code to avoid orphans, the changes on the QueryField is okay for merge.

Reply:
alright, on it!